conf/config.yaml

```diff
@@ -312,15 +272,15 @@ logging:
     #directory: ./logs
     directory: /var/log/__APP__
     # Available variables: .Date for the file date and .Index for different log files on the same day.
+    # Set this to null to disable logging to file.
     #file_name_format: "{{.Date}}-{{.Index}}.log"
     file_name_format: "__LOG_FORMAT__"
     # Date format for file names in the Go time format: https://golang.org/pkg/time/#pkg-constants
-    #file_date_format: 2006-01-02
+    #file_date_format: "2006-01-02"
     # Log file permissions.
-    file_mode: 0600
+    file_mode: 0o600
```

Comment:
I think there is a conflict with this and the systemd.service. 
Here we tell the app to check and fix if needed that de log file is owned by the user running the bridge (default is mautrix_whatsapp) and the perms are 600. 

In the systemd service with redirect output to the same file so it is first created by root on service starting. 
https://github.com/YunoHost-Apps/mautrix_whatsapp_ynh/blob/testing/conf/systemd.service#L9 

I saw with strace that when the bridge stars it loops on something related to the log file (I did not understand all the strace output by the way :sweat_smile: ). I tried to remove the output in the systemd file and it seems to let de bot starts.

Reply:
still not working although erasing the log worked once. After restarting the bridge, it does not start properly. If you manage to solve this permission issue I would be greatful :)
@zamentur which chmod and chown should we set for the bridge logs and how to get the bridge happy with it in config.yaml?